src/6_branch.js

```diff
@@ -1252,12 +1252,19 @@ Branch.prototype['logEvent'] = wrap(callback_params.CALLBACK_ERR, function(done,
 /*** +TOC_ITEM #linkdata-callback &.link()& ^ALL ***/
 Branch.prototype['link'] = wrap(callback_params.CALLBACK_ERR_DATA, function(done, data) {
 	var linkData = utils.cleanLinkData(data);
+	linkData = journeys_utils.trySetBranchViewDataUrls(data);
```

Comment:
Can we change the name of this function? I'm also confused as to why we're modifying the .link() function if the code is supposed to affect the Journeys CTA only?

Reply:
👍